Compatibility fix for go 1.13, flag.Parse() shouldn't be called during
package initialization.
https://golang.org/doc/go1.13#testing
--- a/client_test.go	2020-02-12 14:50:28.991245371 -0500
+++ b/client_test.go	2020-02-12 15:12:37.383523980 -0500
@@ -49,7 +49,6 @@
 	flag.StringVar(&address, "address", defaultAddress, "The address to the containerd socket for use in the tests")
 	flag.BoolVar(&noDaemon, "no-daemon", false, "Do not start a dedicated daemon for the tests")
 	flag.BoolVar(&noCriu, "no-criu", false, "Do not run the checkpoint tests")
-	flag.Parse()
 }
 
 func testContext() (context.Context, context.CancelFunc) {
@@ -59,6 +58,7 @@
 }
 
 func TestMain(m *testing.M) {
+	flag.Parse()
 	if testing.Short() {
 		os.Exit(m.Run())
 	}
